deno-1.24/src/mod.ts

```diff
@@ -0,0 +1,42 @@
+import { sdk } from "./deps.ts";
+
+/*
+  'req' variable has:
+    'headers' - object with request headers
+    'payload' - request body data as a string
+    'env' - object with environment variables
```

Comment:
```suggestion
    'variables' - object with environment variables
```

We need to rename this to variables.. in all the runtimes :)

Reply:
I would love to address that in separate PR when all active PRs are merged. That way I am 100% sure I addressed it everywhere properly - same for class parameters (databaseId) and library versions.

Reply:
okay 👍